src/helpers.ts

```diff
@@ -751,7 +786,7 @@ export function getDiskFullName(disk) {
     if (name === undefined)
         name = disk.device;
 
-    return name;
+    return name || "-";
```

Comment:
This seems odd :thinking: 

Could you add a comment here or in the JSDoc?

Reply:
The idea was to let getDiskFullName always return a string, which is what the rest of the code assumes. However right now things like `disk.source.fil` are all `optString`, since the XML document might or might not have the corresponding element in it. Libvirt might guarantee it, but I haven't bothered to check carefully yet...

I have changed this to return optString.

Reply:
Fair. I meant more just that `-` felt odd as a return as opposed to empty string or something. Means there is a reason for having just `-` and that should've been commented